doc/examples/applications/plot_tool_using_holoviz.py

```diff
@@ -0,0 +1,202 @@
+"""
+==========================================================
+Explore and annotate image data interactively with HoloViz
+==========================================================
+
+In this tutorial, we build interactive tools and an application using
+`Panel <https://panel.holoviz.org>`_ and `HoloViews <https://holoviews.org>`_.
+
+We will build an application based on the Sobel algorithm to interactively
+find a great color map for the visualization. We will add some nice features,
+like a download button, to make sharing your findings easier.

```

Comment:
```suggestion
We build a web application based on the Sobel algorithm which lets the user
find interactively a great color map for the visualization.
Then we add interaction features such as a download button.
```
* Keep the same tense.
* What is "finding a great color map [for the visualization]" supposed to mean? Even if we're not dealing with a scientific problem, the question should be somewhat *interesting*, so the reader can see the point of building this app... From your animation, it looks like the app is only about *changing* the color map interactively (not really *finding*... some kind of optimum, as the verb 'finding' implies). If the app is 'based on the Sobel algorithm' as you write, then what seems 'interesting' would be for the user to vary the `mode` and/or `cval` values...
* I think everyone knows what a download button is for... To make the comment relevant, it should specify what exactly is being downloaded upon clicking the download button in question: a CSV of the full dataset? a CSV of the subset currently displayed? a PNG of the static picture? I think it would be more fitting to specify this when we are at that point in the tutorial.